build.gradle

```diff
@@ -1,74 +1,70 @@
 plugins {
-	id 'fabric-loom' version '0.4-SNAPSHOT'
+	id 'fabric-loom' version '0.10-SNAPSHOT'
+	id 'maven-publish'
+	id "com.github.johnrengelman.shadow" version "7.0.0"
 }
 
-sourceCompatibility = JavaVersion.VERSION_1_8
-targetCompatibility = JavaVersion.VERSION_1_8
+apply plugin: 'java'
+apply plugin: 'fabric-loom'
+apply plugin: 'maven-publish'
+apply plugin: "com.github.johnrengelman.shadow"
+
+defaultTasks('clean', 'build', 'publishToMavenLocal')
+
+sourceCompatibility = '1.16'
+targetCompatibility = '1.16'
 
 archivesBaseName = project.archives_base_name
 version = project.mod_version
 group = project.maven_group
 
+repositories {
+	mavenLocal()
+	maven {url "https://maven.terraformersmc.com/releases"}
+	maven {url "https://maven.shedaniel.me/"}
+	maven {url "https://mvnrepository.com/artifact/org.jsoup/jsoup/"}
+}
+
+configurations {
+	shade
+	compileClasspath.extendsFrom shade
+}
```

Comment:
What is the point of adding the `shade` configuration?

It is not used whatsoever.